src/js/jest.setup.js

```diff
@@ -18,27 +19,34 @@ global.chrome = {
                 version: '1.0.0'
             }
         }),
-        sendMessage: jest.fn()
-    },
-    windows: {
-        create: jest.fn()
+        sendMessage: jest.fn(),
+        onMessage: {
+            addListener: jest.fn()
+        }
     },
     storage: {
+        local: {
+            set: jest.fn().mockImplementation((value) => {
+                return new Promise((resolve) => resolve())
+            })
+        },
         sync: {
             set: jest.fn().mockImplementation((value) => {
                 return new Promise((resolve) => resolve())
             })
         }
     },
-    action: {
-        setBadgeText: jest.fn().mockImplementation((details) => {
-            return new Promise((resolve) => resolve())
-        }),
-        setBadgeBackgroundColor: jest.fn().mockImplementation((details) => {
-            return new Promise((resolve) => resolve())
+    tabs: {
+        query: jest.fn((queryInfo, callback) => {
+            callback([{url: 'http://example.com/story/12345'}])
```

Comment:
I'm debating whether it's better to make sure things work regardless of domain, or whether we should be testing as closely to `app.shortcut.com` which resembles present reality